batch/create-batch-tables.sql

```diff
@@ -86,3 +86,12 @@ CREATE TRIGGER trigger_jobs_update AFTER UPDATE ON jobs
 $$
 
 DELIMITER ;
+
+CREATE TABLE IF NOT EXISTS `batch-attributes` (
+  `batch_id` BIGINT NOT NULL,
+  `key` TEXT(65535) NOT NULL,
+  `value` TEXT(65535),
+  PRIMARY KEY (`batch_id`)
```

Comment:
Do you want the key to be `batch_id`, `key`? Maybe not if you want to grab all the attributes by batch.

Reply:
I want to map from batch_id to its attributes (ergo the key) and from a key-value pair to a batch_id (ergo the index).